run-docker-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+PATH=/usr/local/bin:$PATH
+
+python --version
+
+pip install -q -r requirements.txt
```

Comment:
Why quiet?

Reply:
just to keep the interesting output, and not irrelevant things that pip might want to spit out.

Reply:
The versions of installed requirements seem quite interesting to me